src/main/java/core/basesyntax/service/DailyTransactionsStringToTransactions.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DailyTransactionsStringToTransactions {

```

Comment:
```suggestion
public class CsvFruitTransactionParser implements FruitTransactionParser {
```
To achieve Dependency Inversion and Open Closed you should create abstract types to define behavior
and then create implementations
this way you can add new implementations without changing your existing logic